src/draw/lv_draw_buf.h

```diff
@@ -21,6 +21,9 @@ extern "C" {
  *      DEFINES
  *********************/
 
+/*Use this value to let LVGL calculate stride automatically*/
+#define LV_STRIDE_AUTO 0
+
```

Comment:
To make available in MicroPython too.
```suggestion
LV_EXPORT_CONST_INT(LV_STRIDE_AUTO)
```

cc @PGNetHun

Reply:
It needs a `;` too.

Reply:
Ah, get it. Thanks. Fixed.